src/plugins/inputs/setpass.js

```diff
@@ -0,0 +1,41 @@
+var bcrypt = require("bcrypt-nodejs");
+var Msg = require("../../models/msg");
+
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "setpass") {
```

Comment:
While your argument is true, I don't know how often you have to change your password to warrant having a command for it.